xml/slurm.xml

```diff
@@ -1061,7 +1061,8 @@ success!</screen>
         <title>Convert primary <systemitem class="daemon">slurmdbd</systemitem> first</title>
         <para>
          If a backup database daemon is used, the primary one needs to be
```

Comment:
While we're at it, let's make this as boring, synonym-free, and graspable as possible:
```suggestion
         If a backup <systemitem class="daemon">slurmdbd</systemitem> daemon is used,
         the primary  <systemitem class="daemon">slurmdbd</systemitem> needs to be
```